core/store/src/trie/mem/updating.rs

```diff
@@ -787,6 +857,15 @@ impl<'a, M: ArenaMemory> MemTrieUpdate<'a, M> {
             }
         }
 
+        for i in (0..path.len() - 1).rev() {
+            let node_id = path.get(i).unwrap();
+            let child_id = path.get(i + 1).unwrap();
+            let child_memory_usage = self.get_node_ref(*child_id).memory_usage;
+            let mut node = self.take_node(*node_id);
+            node.memory_usage += child_memory_usage;
```

Comment:
Sounds like we weren't calculating the memory usage here for memtries because of compute_hashes_and_serialized_nodes being called later?

Reply:
We compute it, we just ignore these numbers in that function.